test/jasmine/tests/treemap_test.js

```diff
@@ -629,6 +629,189 @@ describe('Test treemap hover:', function() {
     });
 });
 
+describe('Test treemap hover with levels', function() {
```

Comment:
Can you merge this `describe` block with the other `Test treemap hover` block? The current `run` wrapper should allow you to pass your own `traces` definitions.

Reply:
Combined and added one more test for leaves in https://github.com/plotly/plotly.js/pull/4219/commits/54360be4be39d7d77a8f88fb147909efa6b037d4.

Reply:
Ok.